pkg/controller/pcidevice/pcidevice_controller.go

```diff
@@ -52,16 +52,18 @@ func (h Handler) reconcilePCIDevices(hostname string) error {
 	var setOfRealPCIAddrs map[string]bool = make(map[string]bool)
```

Comment:
We are not deleting removed PCI device objects. If that is the plan, then `setOfRealPCIAddrs` is not needed.

Reply:
We will be deleting those though